Model/DutyCalculator.php

```diff
@@ -213,7 +213,7 @@ public function handleCaseWithCountryAndStateSpecified($address, $apply)
         $quote = $this->checkoutSession->getQuote();
         $this->response->setDuty($quote->getReachDuty());
         $this->handleTaxApplicability($address, $apply);
-        $this->_logger->debug('Country is specified but state is not selected (when both are needed); so we would'.
+        $this->_logger->debug('Country is specified but state is not selected (when both are needed) or the selection of (country, state) combo didnot change; so we would'.
```

Comment:
tiny tiny, "didnot" -> "did not"

Reply:
Commit 148d4bd172dd1829887869ac126b5f2e7f67bc95
Thanks.